lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -10,8 +10,17 @@
 
 const fs = require('fs');
 const path = require('path');
+// @ts-ignore - TODO: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25410
```

Comment:
looks like that landed?
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25410